.github/workflows/release.yml

```diff
@@ -48,7 +48,7 @@ jobs:
     runs-on: [ k8s ]
 
     container:
-      image: docker.repo.orl.eng.hitachivantara.com/pentaho/actions-common:20230823.18
+      image: one.hitachivantara.com/devops-docker-release/pentaho/actions-common:20230823.18
```

Comment:
in [merge.yml](https://github.com/pentaho/actions-common/pull/8/files#diff-bdb37b2ad65e049a0fc043e88813fdcf7ca0118abc11d99bbda676adfcbbb5a1R94), you're referring to `actions-common:20231117.30` but here is something else, why? Do both images exist for sure?

Reply:
Updated the image 
https://github.com/pentaho/actions-common/actions/runs/6903270928